bin/test-bdd.sh

```diff
@@ -4,6 +4,6 @@
 #
 set -ex
 
+ahoy install-site
 echo "==> Run BDD tests"
-ahoy cli "rm -r test/screenshots || true"
 ahoy test-bdd || (ahoy logs; exit 1)
```

Comment:
please remove ahoy logs on this step. it is called in its on discreet step.
this allows you to see the tail instead of just docker log's on what occured.

future state, export to a github action variable and display on the result summary page.
https://rakesh-suryawanshi.medium.com/unit-testing-report-with-github-actions-7216f340044e

Reply:
How about I pipe it to a file and then publish that file as a build artefact?

Reply:
Actually, the workflow already has a step to retrieve logs. Okay, I can just ditch this one, then. Cool.